stable/fairwinds-insights/values.yaml

```diff
@@ -338,10 +338,10 @@ databaseCleanupCronjob:
   resources:
     limits:
       cpu: 500m
-      memory: 1024Mi
+      memory: 2Gi
```

Comment:
I think this was the recommendations cronjob, not database cleanup.

Reply:
🤦

Reply:
FIxed